package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -560,10 +567,16 @@ def _read_next_timestep(self):
 
         triclinic = len(f.readline().split()) == 9  # ITEM BOX BOUNDS
         if triclinic:
-            xlo, xhi, xy = map(float, f.readline().split())
-            ylo, yhi, xz = map(float, f.readline().split())
+            xlo_bound, xhi_bound, xy = map(float, f.readline().split())
+            ylo_bound, yhi_bound, xz = map(float, f.readline().split())
             zlo, zhi, yz = map(float, f.readline().split())
 
+            # converts orthogonal bounding box to the conventional format, see https://docs.lammps.org/Howto_triclinic.html
```

Comment:
Can you quickly address the two PEP8 issues if possible? https://github.com/MDAnalysis/mdanalysis/pull/3403#issuecomment-913989898

Ignore the ones related to datafiles, that file is a lost cause.

In this case just splitting over two lines would be fine.